common/views/components/Caption/index.tsx

```diff
@@ -19,7 +19,6 @@ const CaptionText = styled(Space).attrs({
 `;
 
 const CaptionWrapper = styled.div`
-  overflow: hidden;
```

Comment:
I'm not sure why this was `overflow: hidden` but it can't be if we want to show the hover cards. I couldn't see any problems visually with it being removed